xontrib/fzf-widgets.xsh

```diff
@@ -6,11 +6,12 @@ from xonsh.completers.path import complete_path
 from xonsh.platform import ptk_shell_type
 from prompt_toolkit.keys import Keys
 
-__all__ = ()
+__all__ = ['Keys']
```

Comment:
I don't really like this change. I don't want to pollute the global namespace.

Reply:
Btw, you don't really need to use the `Keys` enum to bind the keyboard shortcuts. You can use the respective string representation as well, e.g. `$fzf_history_binding = 'c-r'` instead of `Keys.ControlR`.

Reply:
Thanks for the reply. I tested it and it seems to work. If you are interested, I will fix this in a day or two and push this.

Reply:
Yes, please. I'll be glad to merge the rest in. Thank you!